### DISCUSSION TITLE
question about "Block Outsider Intrusion into LAN"

### DISCUSSION THREAD

**Discussion by git70** [uBlockOrigin/uBlock-issues#3344] [General]
Hi

I have the `Block Outsider Intrusion into LAN` list enabled and it seems to me that a few days ago it was updated with the threat [0.0.0.0 Day](https://www.oligo.security/blog/0-0-0-0-day-exploiting-localhost-apis-from-the-browser) 

According to [this test](http://rebind.it:8080/scan-manager.html)
- uBO shows 0.0.0.0 blocked 
- [Behave](https://github.com/mindedsecurity/behave) did not react 
- [Port_Authority](https://github.com/ACK-J/Port_Authority) did not react

Since both extensions seem unmaintained I intend to abandon them, but I have a question does uBO: 
- block all possible types of port scanning? (HTTP / HTTPS / WS / WSS / FTP / FTPS)
- does it block Websocket?
- does it protect against DNS Rebinding?
- does it protect against TOCTOU ([whatever that is](https://github.com/mindedsecurity/behave#dns-resolution-to-private-ips))

I just want to make sure that the above `Behave` and `Port_Authority` extensions do nothing more than uBO and I can remove them to have fewer add-ons.

Additionally, I am not sure how to interpret [this test](https://samy.pl/webscan/)
uBO shows blocked requests, but the test shows "found host"

Can someone help me?

**Comment by garry-ut99** [comment]
I don't know whether you've seen these threads or not, also I don't guarantee that you will find all answers, but perhaps some answers:
- https://github.com/uBlockOrigin/uBlock-issues/issues/1070
- https://github.com/uBlockOrigin/uAssets/issues/4318
- https://github.com/easylist/easylist/issues/19041

**Comment by stephenhawk8054** [comment]
Use [uBO's logger](https://github.com/gorhill/uBlock/wiki/The-logger) and check which connections are being blocked
